misc/scripts/test-utils

```diff
@@ -8,6 +8,7 @@ declare -A util_vars=(
   [is_cheated]=0
   [is_installed]=0
   [non_interactive]=0
+  [use_threads]=1
```

Comment:
I don't know how this affects the behavior of the generated Makefile from the test script, but from a practical point of view I would prefer for parallelism to be opt-in. It would be preferable that the default option is not to starve the system resources, especially if running the tests on a server shared by others.

Can we not replicate the Makefile interface for this, with the `-j` flag and an argument to it?